src/intents.js

```diff
@@ -21,7 +21,7 @@ const errorSerializer = (() => {
 })()
 
 // inject iframe for service in given element
-function injectService (url, element, intent, data, onReadyCallback) {
+function injectService (url, element, intent, data, onReadyCallback, title) {
```

Comment:
A few things here:
* To pass a `title` parameter here, we will need to pass it to the method `start` line 125.
* After `onReadyCallback`, the parameter list is beginning to grow. I suggest to start using on `options` parameter as last parameter. We need to keep the first ones `url`,  `element`,  `intent`, and `data` as they are all mandatory and important. So now using an `options` parameter now forces us to maintain legacy a little time. This `options` parameter should also be passed to the `start` method l. 125.
* The `title` parameter should not be mandatory, however it is a really nice thing to have one, if it improves accessibility. So we should set a default title.

I suggest to start this method this way now :
```js
const defaultTitle = `Cozy intent` // Should we avoid i18n for a default title ? 

[...]

function injectService  (url, element, intent, data, options = {}) {
  const isLegacyOnReadyCallback = typeof options === 'function'

  if (isLegacyOnReadyCallback) {
    console.warn && console.warn('onReadyCallback is now deprecated.')
  }

  const onReadyCallback = isLegacyOnreadyCallback ? options : options.onReadyCallback
  const title = options.title || defaultTitle
  [...]
```

What to you think ? (ping also @CPatchane and @y-lohse)

Reply:
I just add a 
`  // TODO: implement 'title' attribute` for now, because it's gonna take some time for me to add this feature, but I keep it in mind to do it any time in the near future.